exercises/triangle/triangle.go

```diff
@@ -2,6 +2,8 @@
 
 package triangle
 
+const TestVersion = 1
```

Comment:
Notice that most other elements of this file (`KindFromSides`, `Kind`, `Equ`, etc...) have some sort of explanatory comment. I chose not to include one for `TestVersion` since students should have gotten the drill already from previous exercises. Let me know if you disagree.